allowance_unece/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Decodio Applications (<http://decod.io>)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Base UBL allowance",
+    'version': "12.0.1.0.0",
+    "author": "Decodio Applications, Odoo Community Association (OCA)",
+    "development_status": "RC",
```

Comment:
Here are the allowed values for `development_status`:

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L8